aspnetcore/tutorials/signalr.md

```diff
@@ -480,4 +485,8 @@ The SignalR server must be configured to pass SignalR requests to SignalR.
 >   dotnet dev-certs https --trust
 >   ```
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

Comment:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

Reply:
@wadepickett  why all the commits and suggestions? Can't you fix everything in one commit?

Reply:
@Rick-Anderson I can.  If this is a big thing for you I can change the way I work.  I usually work from a local clone.  Today I had a family member going to the hospitaL.  I ended up having to quickly work out of gitHub which I don't usually do, and so today it was different.

Reply:
>  I ended up having to quickly work out of gitHub which I don't usually do, and so today it was different.

It's much faster and more efficient to edit it the browser and do a global replace, then commit. Less work, less noise.

Reply:
![image](https://user-images.githubusercontent.com/3605364/164117693-2fd73fb2-bc14-47c2-b11e-16637a585400.png)